src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -541,7 +541,8 @@ class ContentSidebar extends PureComponent<Props, State> {
                 const assignment = assignments.entries.find((a) => a.id === item.id);
                 if (assignment) {
                     return {
-                        ...assignment
+                        ...assignment,
+                        resolution_state: assignment.message.toLowerCase() || assignment.resolution_state
```

Comment:
Are we calling this method on read, update, and eventually create? Also, when does messages become populated vs. resolution state? Is it only there after updates. It would be great if we could look at one field.